src/Controller/FavoritesController.php

```diff
@@ -115,14 +145,14 @@ public function add(Request $request) {
 
     $favorite_item = $this->entityTypeManager()
       ->getStorage('vy_favorite_item')
-      ->create([
-        'ref_entity_type' => $data['entity_type'],
-        'ref_entity_bundle' => $data['entity_bundle'],
-        'ref_entity_id' => $data['entity_id'],
-      ]);
+      ->create($data);
```

Comment:
```
fetch("http://carnation.pr-231.openy-gated-content.sandbox-2.ci.fivejars.com/api/favorites/add?_format=json", {
  "headers": {
    "accept": "application/json, text/plain, */*",
    "accept-language": "ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7,uk;q=0.6",
    "content-type": "application/json"
  },
  "referrer": "http://carnation.pr-231.openy-gated-content.sandbox-2.ci.fivejars.com/virtual-ymca",
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": "{\"uid\":16,\"ref_entity_type\":\"node\",\"ref_entity_bundle\":\"gc_video\",\"ref_entity_id\":3}",
  "method": "POST",
  "mode": "cors",
  "credentials": "include"
});
```
Allowed me to "like" the item for another user. Let's return to the previous implementation, or, at least, unset the `uid` from an input data.

Reply:
Fixed in https://github.com/fivejars/openy_gated_content/pull/231/commits/9289916fa04cd0719ecd5870623170c62ed8cf87